### PR TITLE
Docs for "new" functions and other improvements

### PR DESCRIPTION
**Pull request by wlsf82**
Use the commit messages for helping in the review process and review commit by commit.

## Pull Request Checklist

- [x] Have you followed the steps in our [Contributing](../../blob/master/docs/CONTRIBUTING.md#steps-to-contribute) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?
- [x] Have you successfully ran tests with your changes locally?
- [x] Have you written new tests for your changes, if necessary?
- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [x] Have you made improvements to the documentation, if necessary? ([docs/](../../blob/master/docs/) folder and [README.md](../../blob/master/README.md))
- [x] Have you linked the correct [GitHub issue](../../issues), if there is any?

> Pull requests that do not pass the automated tests on [SemaphoreCI](http://semaphoreci.com) and the code quality verification on [Better Code Hub](https://bettercodehub.com/) will not be reviewed

### CODE REVIEW

File: docs/EXAMPLES.md [hunk 1]
```diff
@@ -244,6 +244,30 @@ describe("foo", () => {
 });
 ```
 
+> Note: this function will be deprecated in favor of the function `click`.
```
Comment (PauloGoncalvesBH) [inline docs/EXAMPLES.md]: This 'Note' looks better below the method title that will be depreciated.

File: docs/EXAMPLES.md [hunk 2]
```diff
@@ -271,6 +295,30 @@ describe("foo", () => {
 });
 ```
 
+> Note: this function will be deprecated in favor of the function `fillFieldWithText`.
```
Comment (PauloGoncalvesBH) [inline docs/EXAMPLES.md]: This 'Note' looks better below the method title that will be depreciated.

File: docs/EXAMPLES.md [hunk 3]
```diff
@@ -302,6 +353,41 @@ describe("foo", () => {
 });
 ```
 
+> Note: this function will be deprecated in favor of the function `uploadFileIntoInputField`.
```
Comment (PauloGoncalvesBH) [inline docs/EXAMPLES.md]: This 'Note' looks better below the method title that will be depreciated.

File: docs/EXAMPLES.md [hunk 4]
```diff
@@ -329,6 +415,31 @@ describe("foo", () => {
 });
 ```
 
+> Note: this function will be deprecated in favor of the function `clear`.
```
Comment (PauloGoncalvesBH) [inline docs/EXAMPLES.md]: This 'Note' looks better below the method title that will be depreciated.

File: docs/EXAMPLES.md [hunk 5]
```diff
@@ -356,6 +467,30 @@ describe("foo", () => {
 });
 ```
 
+> Note: this function will be deprecated in favor of the function `clearFieldAndFillItWithText`.
```
Comment (PauloGoncalvesBH) [inline docs/EXAMPLES.md]: This 'Note' looks better below the method title that will be depreciated.

File: docs/EXAMPLES.md [hunk 6]
```diff
@@ -384,6 +519,32 @@ describe("foo", () => {
 });
 ```
 
+> Note: this function will be deprecated in favor of the function `tap`.
```
Comment (PauloGoncalvesBH) [inline docs/EXAMPLES.md]: This 'Note' looks better below the method title that will be depreciated.

File: docs/EXAMPLES.md [hunk 7]
```diff
@@ -575,6 +736,34 @@ describe("foo", () => {
 });
 ```
 
+> Note: this function will be deprecated in favor of the function `fillFieldWithTextAndPressEnter`.
```
Comment (PauloGoncalvesBH) [inline docs/EXAMPLES.md]: This 'Note' looks better below the method title that will be depreciated.

File: docs/EXAMPLES.md [hunk 8]
```diff
@@ -601,4 +790,28 @@ describe("foo", () => {
 });
 ```
 
+> Note: this function will be deprecated in favor of the function `scrollToElement`.
```
Comment (PauloGoncalvesBH) [inline docs/EXAMPLES.md]: This 'Note' looks better below the method title that will be depreciated.

File: test/protractor.conf.js [hunk 9]
```diff
@@ -11,6 +11,7 @@ module.exports.config = {
   specs: ["*.spec.js"],
   onPrepare() {
     browser.waitForAngularEnabled(false);
+    // afterEach(() => browser.sleep(1000));
```
Comment (PauloGoncalvesBH) [inline test/protractor.conf.js]: We're using 'headless', this isn't useful for us, Why keep?

File: test/protractor.conf.js [hunk 10]
```diff
@@ -19,4 +20,5 @@ module.exports.config = {
     includeStackTrace: true,
     random: true
   }
+  // highlightDelay: 1000
```
Comment (PauloGoncalvesBH) [inline test/protractor.conf.js]: We're using 'headless', this isn't useful for us, Why keep?

File: docs/EXAMPLES.md (hunk 1 above)
Comment (wlsf82) [reply docs/EXAMPLES.md]: When debugging I comment the headless config and this turns to be useful.

**Comment by PauloGoncalvesBH** [comment]
Is necessary to implement (need see if in that or other PR) deprecating warning on console and tags on the code.
 A good example: https://css-tricks.com/approaches-to-deprecating-code-in-javascript/

**Comment by PauloGoncalvesBH** [comment]
This PR is the solution of the issue #26. Only want deprecation warning and define a transitional period.

File: docs/EXAMPLES.md (hunk 1 above)
Comment (wlsf82) [reply docs/EXAMPLES.md]: Good point. I'll update.

File: test/protractor.conf.js (hunk 9 above)
Comment (wlsf82) [reply test/protractor.conf.js]: When debugging I comment the headless config and this turns to be useful.

File: test/protractor.conf.js (hunk 10 above)
Comment (wlsf82) [reply test/protractor.conf.js]: When debugging I comment the headless config and this turns to be useful.

**Comment by wlsf82** [comment]
@PauloGoncalvesBH, we can add the deprecation warnings in a future PR.

**Comment by wlsf82** [comment]
Back to you, @PauloGoncalvesBH.

**Comment by PauloGoncalvesBH** [comment]
I added a new suggestion @wlsf82

**Comment by wlsf82** [comment]
@PauloGoncalvesBH, I have simplified the `js` file of the sample app, and now the sensitive data is not needed anymore. By the way, I'm revoking that key. Back to you.

**Comment by wlsf82** [comment]
Hey @PauloGoncalvesBH, any comments? If not, can you approve it, please?

**Comment by PauloGoncalvesBH** [comment]
I tried to merge, but is required 1 approving review. How I approve the PR?

**Pull request merged**